### ISSUE TITLE
[NOT A BUG] ./setup.sh not running

### ISSUE THREAD

**Issue by lukematthews** [docker-mailserver/docker-mailserver#2211]
### Miscellaneous first checks

- [X] I checked that all ports are open and not blocked by my ISP / hosting provider.
- [X] I know that SSL errors are likely the result of a wrong setup on the user side and not caused by DMS itself. I'm confident my setup is correct.

### Affected Component(s)

./setup.sh not running

### What happened and when does this occur?

```Markdown
Hi All,

First time user starting with a fresh install.

I've downloaded the docker-compose.yml and mailserver.env in the root of the repo. Modifying mailserver.env to enable LDAP let's the service start by running:
`docker-compose up --build -d`

Running `./setup.sh help` gives me:

`OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "setup": executable file not found in $PATH: unknown`
```

### What did you expect to happen?

```Markdown
Well, anything. Passing `help` I don't even get the help messages.
```

### How do we replicate the issue?

```Markdown
wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/setup.sh
wget https://github.com/docker-mailserver/docker-mailserver/blob/master/docker-compose.yml
wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/mailserver.env
chmod a+x setup.sh

modified mailserver.env to enable LDAP 
ENABLE_LDAP=1

./setup.sh
```

### DMS version

latest

### How much RAM is available to DMS explicitly?

more than 4GB

### How many CPU cores are available?

more than 4 Cores

### Is DMS running in a virtualized environment?

... a physical host (with real, physical CPU cores, i.e. bare metal)

### What operating system is DMS running on?

Linux

### What instruction set architecture is DMS running on?

x86_64 / AMD64

### I/O - Persistent memory

Standard EXT4 setup

### What container orchestration tool are you using?

Docker Compose

### Docker version

Docker version 20.10.8, build 3967b7d

### Docker Compose version

docker-compose version 1.29.2, build 5becea4c

### The output of `uname -a`

Linux home-server 5.4.0-86-generic #97-Ubuntu SMP Fri Sep 17 19:19:40 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

### What level of experience do you have with Docker and mail servers?

I'm comfortable with Docker / containers, but I'm relatively new to all this mail stuff

### Code of conduct

- [X] I have read this project's [Code of Conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md) and I agree
- [X] I have read the [README](https://github.com/docker-mailserver/docker-mailserver/blob/master/README.md) and the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/) and I searched the [issue tracker](https://github.com/docker-mailserver/docker-mailserver/issues?q=is%3Aissue) but could not find a solution

**Comment by georglauterbach** [comment]
(Very very obvious) Duplicate of #2203. @lukematthews you stated that you searched the issue tracker with your last tick in the form. #2203 _**is even pinned**_. Sorry, but I cannot believe you really searched this tracker for anything… tell me, why should we bother helping with your issues now?

**Comment by lukematthews** [comment]
Sorry about that. You're right, after submitting this, I went through the closed as opposed to just the open and saw a couple of bugs there already.

Again, I'm sorry about that.

**Comment by lukematthews** [comment]
and... moved to the right tag. Works. Problem with being a software developer is that reading is not a strength!

**Comment by polarathene** [comment]
> moved to the right tag

Assuming `:edge` (build of last development commit), note that this differs from `:latest` (active stable release). For similar reasons to why you shouldn't rely on `:latest` in production, `:edge` is risky. It is not guaranteed to be in a known good condition and may experience missed bugs that we try to catch prior to making a new release. Think of `:edge` as a beta version.

Both tags may result in breaking changes if you rely on them, as we may release new major versions and since these tags are not immutable, you could accidentally pull an update without knowing, and not having reviewed any changelog notes as a result. Depending on changes made, that could risk breakage, bugs or (hopefully unlikely) data loss.

---

The issue you experienced unfortunately is due to the upcoming 10.2 version release preparation, we are removing this problem with needing to manually download a release version specific copy of `setup.sh`. [The README does warn the reader about this](https://github.com/docker-mailserver/docker-mailserver#setupsh-for-docker-mailserver-version-v101x-and-below).

**Comment by polarathene** [comment]
Leaving this alternatively pinned issue open until the 10.2 release. It should at least appear on the 1st page of issues for now (_despite the obvious pins at the top that may be missed_).
